spec/controllers/bulk_ingest_controller_spec.rb

```diff
@@ -60,38 +60,62 @@
   end
 
   describe "POST #browse_everything_files" do
+    let(:attributes) do
+      {
+        workflow: { state: "pending" },
+        collections: ["1234567"],
+        visibility: "open",
+        mvw: false,
+        selected_files: selected_files
+      }
+    end
+    let(:selected_files) do
+      {
+        "0" => {
+          "url" => "file:///base/4609321/1.tif",
+          "file_name" => "1.tif",
+          "file_size" => "100"
+        }
+      }
+    end
+
     before do
       allow(IngestFolderJob).to receive(:perform_later)
       allow(IngestFoldersJob).to receive(:perform_later)
     end
 
     context "with one single-volume resource" do
```

Comment:
Can you update this and/or the test name to mention the bib id dir? To set it in context against line 105.

Reply:
👍